build.sbt

```diff
@@ -124,7 +124,6 @@ lazy val kcl = projectMatrix
   .dependsOn(shared, `kinesis-client`)
 
 lazy val `kcl-http4s` = projectMatrix
-  .enablePlugins(Smithy4sCodegenPlugin)
```

Comment:
This one needs to stay because we are referencing the generated classes from it.

Reply:
The plugin shouldn't be needed for accessing generated classes, I'll have to dive deeper to see why they're not available there :|

Reply:
AH. this module has smithy files itself, I see.